debian/copyright

```diff
@@ -0,0 +1,52 @@
+
+Format: http://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
+Upstream-Name: paflib
+Source: https://github.com/leitao/paflib_deb
```

Comment:
According to debian policy, this field should have "an explanation of where the **upstream** source came from" [1], thus, I believe that you should change this line to:

     Source: https://github.com/paflib/paflib

[1] https://www.debian.org/doc/packaging-manuals/copyright-format/1.0/#source-field